packages/components/psammead-section-divider/README.md

```diff
@@ -0,0 +1,60 @@
+# psammead-section-divider - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-section-divider%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-section-divider%2Fpackage.json) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/section-divider--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-section-divider.svg)](https://www.npmjs.com/package/@bbc/psammead-section-divider) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `@bbc/psammead-section-divider` package exports a single Section Divider component.
```

Comment:
This is a comment about the naming of this component within Psammead.  Can it please be renamed as `@bbc/psammead-divider`? This is because the use of `section-divider` appears to refer to the HTML semantic `<section>`, which is not the case here.

Reply:
I had a catch up with UX about this component yesterday, they are now referring to this component as a 'Section Label'. Would it be best to rename this component to align as closely as possible with their name? This would also have the issue that `section-label` might be referring to `<section>`, but `label` also feels very vague.